superset/db_engine_specs/bigquery.py

```diff
@@ -422,6 +422,11 @@ def _get_client(
                 "Could not import libraries needed to connect to BigQuery."
             )
 
+        if not engine.dialect.credentials_info:
+            raise SupersetDBAPIConnectionError(
+                "The database credentials do not contain a 'credentials_info' field."
+            )
+
```

Comment:
I double checked our docs and confirmed that it instructs to add the credentials JSON to the connection setup, but I believe it's also possible to run Superset with the JSON set in the `GOOGLE_APPLICATION_CREDENTIALS` env variable (haven't tested it, but I believe if you import a connection with no creds it tells it couldn't find in this env variable too).

I wonder if we currently have users running it that way, and if so if this should be added as a note to the `UPDATING.md` file?

Reply:
Ah, interesting. I think we should then check that `GOOGLE_APPLICATION_CREDENTIALS` is also not set before raising the exception.